en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -1,7 +1,62 @@
 # Silverstripe CMS 5.0.0-beta1
 
-## Post CMS5 beta freeze
-One of the primary purposes of the beta testing is to give third party maintainers and developers an opportunity to upgrade their modules to be compatible with CMS5. Once the CMS5 beta is tagged, our aim is to:
+This is the first beta release for Silverstripe CMS 5. This is a pre-release version of Silverstripe CMS 5. It is not suitable for a production environment.
+
+We are anticipating that we will tag a Silverstripe CMS 5 Release Candidate in March 2023 and a stable release in April 2023.
+
+## Overview
+- [About the Beta phase](#about-the-beta-phase)
+  - [Trying the beta](#trying-the-beta-trying)
+  - [Reporting bugs](#reporting-bugs)
+  - [Change freeze](#change-freeze)
+  - [Known issues](#known-issues)
+- [Upgrading your third party module](#upgrading-your-third-party-module)
+  - [Planning your upgrade](#planning-your-upgrade)
+  - [Stop using deprecated code](#stop-using-deprecated-code)
+  - [Upgrade your constraints](#upgrade-your-constraints)
+  - [Ship a Silverstripe CMS 5 version of your module](#ship-a-silverstripe-cms-5-version-of-your-module)
+- [Change to commercially supported modules](#supported-modules)
+- [Fixed dependencies](#fixed-depdencies)
+- [API changes](#api-changes)
+  - [Removed API (by module, alphabetically)](#api-removed)
+  - [General changes](#api-general)
+  - [Email](#api-email)
+  - [ORM](#api-orm)
+- [Dependency changes](#dependency-changes)
+  - [General changes](#dep-general)
+  - [TinyMCE upgraded from 4 to 6](#tinymce6)
+- [Front-end build stack upgrades](#front-end)
+  - [Webpack config](#webpack-config)
+  - [Updated javascript dependencies](#updated-javascript-dependencies)
+  - [Common upgrade patterns](#common-upgrade-patterns)
+  - [Breaking changes in _our_ javascript components or API](#js-api)
+- [Other Breaking Changes](#other-breaking-changes)
+  - [URLs generated by Silverstripe CMS don't include a trailing slash by default](#trailing-slash)
+  - [CWP Agency Extensions](#cwp-agency-extensions)
+- [Features and enhancements](#features-and-enhancements)
+  - [Other features](#other-features)
+- [Bugfixes](#bugfixes)
+
+
+## About the Beta phase
+
+### Trying the beta
+
+Creating a Silverstripe CMS 5 test project is just as easy as before. Simply run this command to try out the Silverstripe CMS 5 beta.
+
+```
+composer create-project silverstripe/installer my-site 5.0.0-beta1
+```
+
+## Reporting bugs
```

Comment:
We should also encourage people to create pull requests (linked to the relevant issue, which they should create first if it doesn't already exist) to resolve regressions - and we should link to the contributing guidelines.

Reply:
I've added a blurb to that affect at the end.